docs/data/projects.yml

```diff
@@ -589,3 +589,9 @@
   gh: MarshalX/python-webrtc
   ci: [github]
   os: [windows, apple, linux]
+
+- name: cf-units
+  gh: SciTools/cf-units
+  ci: [github]
+  os: [apple, linux]
+  notes: Units of measure as required by the Climate and Forecast (CF) metadata conventions
```

Comment:
This will pick up the repo description if not given, so if we have a notes, it's often wheel specific.

Reply:
Thanks @henryiii 

We do have a description on the `cf-units` GH repo:

![image](https://user-images.githubusercontent.com/2051656/178305558-6476070f-2f35-474d-a8e8-2c63d6a87986.png)

Are you happy for me to remove the `notes` entry in the `docs/data/projects.yml` file?

Reply:
Done 👍